docs/commit-hygiene.md

```
@@ -0,0 +1,111 @@
+### 6. Build Before Testing
+
+Some packages require building before tests will pass:
+
+```bash
+yarn build && yarn test
+```
+
+- client-utils
+- cosmic-proto
+
+Issue: [Changes to package "client-utils" are not visible without compilation #11954
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside link text

(MD039, no-space-in-links)